corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -243,6 +245,39 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             };
         },
 
+        iconClick: function (e) {
+            e.stopImmediatePropagation();
+            const origin = window.location.origin;
+            const currentUrlToObject = formplayerUtils.currentUrlToObject();
+            const user = FormplayerFrontend.getChannel().request('currentUser');
+            const appId = currentUrlToObject.copyOf;
+            const domain = user.domain;
+            const caseId = this.model.get('id');
+            const urlTemplate = this.options.endpointActions[1]['urlTemplate'];
+            const actionUrl = origin + urlTemplate
+                .replace("{domain}", domain)
+                .replace("{appid}", appId)
+                .replace("{case_id}", caseId);
+            this.iconSmartLink(e, actionUrl);
```

Comment:
At the moment cases are not being updated with this call to ` this.iconSmartLink`, but if you copy `actionUrl` and paste it in the browser the form will be submitted and the case will be correctly updated. Looks like ajax request needs something else.